### DISCUSSION TITLE
Weather app coloring isn't accurate

### DISCUSSION THREAD

**Discussion by 88572** [InfiniTimeOrg/InfiniTime#2029] [General]
If I understand the weather PR right, colder temperatures should be closer to blue. However, this doesn't work the way I would expect.
![IMG_20240225_140743.jpg](https://github.com/InfiniTimeOrg/InfiniTime/assets/82726593/997801a7-7015-4366-bc0d-aabbf04f67f2)

This may just be a matter of personal opinion, but anything below 50°F feels cold.

**Comment by FintasticMan** [comment]
In the end, we decided to say that less than or equal to 32°F would be blue, less than or equal to 39.2°F would be cyan, less than 80.6°F would be yellow, and higher would be orange. These values were chosen a bit arbitrarily, but it's very difficult to choose neutral values. This is because climates are so different around the world, so something that is cold for you might be warm somewhere else, or vice-versa. I would be up for changing it though if we have a good solution to this problem.
